Xamarin.ExposureNotification/ExposureNotification.ios.cs

```diff
@@ -160,11 +161,54 @@ static async Task<(ExposureDetectionSummary, Func<Task<IEnumerable<ExposureInfo>
 			var c = await GetConfigurationAsync();
 			var m = await GetManagerAsync();
 
-			var detectionSummary = await m.DetectExposuresAsync(
+			// Extract all the files from the zips
+			var allFiles = new List<string>();
+			foreach (var file in keyFiles)
+			{
+				using var stream = File.OpenRead(file);
+				using var archive = new ZipArchive(stream, ZipArchiveMode.Read);
+
+				// .bin
+				var binTmp = Path.Combine(FileSystem.CacheDirectory, Guid.NewGuid().ToString() + ".bin");
```

Comment:
Maybe we should use the same guid for both the .bin and .sig? This could make debugging a bit more sane at the very least.